admin_guide/managing_projects.adoc

```diff
@@ -94,3 +101,55 @@ or:
 To request a new project, fill out the project request form located at
 https://internal.example.com/openshift-project-request.
 ----
+
+== Node Selectors
+
+Node selectors are used in conjunction with labels to control pod placement. 
+
+[NOTE]
+Labels can be assigned link:../install_config/install/advanced_install.html#configuring-node-host-labels[during an advanced installation], or link:../admin_guide/manage_nodes.html#updating-labels-on-nodes[added to a node after installation].
+
+Using node selectors, you can assign a pod or set of pods to specific nodes, or limit a project to only run on a desired set of nodes. For example, if you have a OpenShift topology with multiple regions, node selectors help restrict specific OpenShift projects to only deploy pods into a specific region. 
+
+To create a project with a node selector, use the `--node-selector` option. The following creates a new project named `myproject`, but runs only on the nodes labeled `user-node` and `east`:
+
+====
+----
+$ oadm new-project myproject \
+    --node-selector='type=user-node,region=east'
+----
+====
+
+[NOTE]
+While the `new-project` option is available for both `oadm` and `oc`, the cluster administrator and developer commands respectively, creating a new project with a node selector is only available with the `oadm` command. The `new-project` option is not available to project developers when self-provisioning projects.
+
+Using the above command adds an `annotation` section to the project configuration:
+
+====
+----
+...
+metadata:
+  annotations:
+    openshift.io/node-selector: type=user-node,region=east
+...
+----
+====
+
+Alternatively, edit the *_/etc/origin/master/master-config.yaml_* master configuration file and add a node selector value. This applies the node selector to pods created in all projects without a specified nodeSelector value:
+
+====
+----
+...
+projectConfig:
+  nodeSelector: "type=user-node,region=east"
+...
+----
+====
+
+Ensure to restart the master OpenShift service for the changes to take effect:
+
+====
+----
+$ systemctl restart atomic-openshift-master
```

Comment:
This requires root privileges, so change the $ prompt to a # to signal it should be run as root (or as non-root w/ sudo).